kie-drools-wb-parent/kie-drools-wb-webapp/src/main/resources/security-policy.properties

```diff
@@ -44,3 +48,10 @@ default.permission.project.release=false
 default.permission.repository.configure=false
 default.permission.planner.available=false
 default.permission.globalpreferences.edit=false
+
+# See https://issues.jboss.org/browse/GUVNOR-3524
+default.permission.perspective.read.ProvisioningManagementPerspective=true
+default.permission.editor.read.GuidedDecisionTreeEditorPresenter=true
+default.permission.editor.read.GuidedScoreCardEditor=true
+default.permission.editor.read.ScoreCardXLSEditor=true
+default.permission.editor.read.BPMNDiagramEditor=true
```

Comment:
@manstis  I think this is related to my comment with video?

Reply:
@jomarko Yes, these are the _Community_ defaults. The _Product_ defaults are [here](https://github.com/kiegroup/kie-wb-distributions/pull/631/files#diff-d1e07265f5d3d91c9e0ec6dc3f7a5b6fR52) and [here](https://github.com/kiegroup/kie-wb-distributions/pull/631/files#diff-c762b0be4f563278c01f22ac07fdab2cR53)